org.osgi.test.cases.typedevent/bnd.bnd

```diff
@@ -9,6 +9,9 @@ Import-Package: ${-signaturetest}, *
 -buildpath: \
 	org.osgi.test.support;version=project,\
 	org.osgi.service.typedevent;version=latest,\
+	org.osgi.resource;version=latest,\
```

Comment:
Please use actual version numbers for dependent APIs. Preferable the oldest version that satisfies your needs. Only the API under test should use version=latest.

The point is that the test jar should not impose versions higher than what is needed.

Also, these should all be `maven-scope=provided` since they are not needed at runtime or are provided by the OSGi framework impl.